versions.lock

```diff
@@ -16,11 +16,12 @@ com.palantir.safe-logging:safe-logging:1.8.2 (3 constraints: 422aeb21)
 com.squareup.okhttp3:okhttp:3.9.0 (1 constraints: 0e051536)
 com.squareup.okio:okio:1.13.0 (1 constraints: 7f0cb509)
 io.undertow:undertow-core:2.0.17.Final (1 constraints: 54072861)
+jakarta.servlet:jakarta.servlet-api:4.0.4 (1 constraints: 0a050336)
+jakarta.ws.rs:jakarta.ws.rs-api:2.1.6 (1 constraints: 0b05fe35)
 javax.annotation:javax.annotation-api:1.2 (2 constraints: 2d21193d)
 javax.inject:javax.inject:1 (2 constraints: d614a0ab)
-javax.servlet:javax.servlet-api:3.1.0 (2 constraints: 88129b22)
 javax.validation:validation-api:1.1.0.Final (3 constraints: dc393f20)
-javax.ws.rs:javax.ws.rs-api:2.0.1 (8 constraints: ed9109e8)
+javax.ws.rs:javax.ws.rs-api:2.0.1 (7 constraints: e98c8a03)
```

Comment:
I don't think this should still be here as it should be replaced by `jakarta.ws.rs:jakarta.ws.rs-api` above, right?

Reply:
you're right, let me look deeper into this

Reply:
https://github.com/palantir/tracing-java/pull/565 for a fix for this.